package.json

```diff
@@ -21,6 +21,7 @@
     "moment": "2.30.1",
     "noty": "3.2.0-beta",
     "oidc-client-ts": "3.1.0",
+    "outdated-browser-rework": "^3.0.1",
```

Comment:
This package was last published 4 years ago and depends on another package that had multiple major revisions (and potentially security fixes).

I think we should evaluate whether we can do this without depending on an external package, and use our existing notification machinery to warn users.

Reply:
Agreed on this. Additionally, we generally pin versions explicitly due to historic problems with npm point updates.

Reply:
I could probably do it using the Alert already in Duos, figuring out how to detect the browser being used will be the harder part - I'll look into it

Reply:
@fboulnois @rushtong After some searching, I found an alternative approach could be to detect the browser using the user agent but this seems generally to be recommended **against** for being unreliable. I'm not sure if this would be better or worse than using this package, and I can't seem to find other straightforward ways to do browser detection. Let me know your thoughts!

Reply:
This looks promising as an alternative: https://github.com/pure-js/browser-detection